src/models/config.py

```diff
@@ -62,6 +63,7 @@ def _ini_reader(self, file: str) -> None:
         self.item_ids = config["MAIN"].get("ItemIDs").split(
             ',') if "ItemIDs" in config["MAIN"] else []
         self.sleep_time = config["MAIN"].getint("SleepTime")
+        self.schedule_cron = config["MAIN"].getint("ScheduleCron")
```

Comment:
This should be `.get("ScheduleCron")`, since it is a string?

Reply:
Correct. Fixed now [78b408b](https://github.com/zinen/tgtg/commit/78b408b71bed3196735efd82a0c0b4866a375d1a)